build/localmode/RunSamples.cmd

```diff
@@ -13,6 +19,13 @@ if "%1" == "" (
         set VERBOSE="verbose"
         @echo [RunSamples.cmd] VERBOSE is !VERBOSE!
     )
+
```

Comment:
One option is to keep the change in RunSamples for development benefits, but not offering RunSamples in the wiki page for running examples. So in the future, if we offer another script for running example, it will not introduce confusions,